src/Aquifer.Data/Entities/ResourceContentVersionSimilarityScore.cs

```diff
@@ -0,0 +1,22 @@
+namespace Aquifer.Data.Entities;
+
+public class ResourceContentVersionSimilarityScore
+{
+    public int Id { get; set; }
+    public int BaseVersionId { get; set; }
+    public ResourceContentVersionTypes BaseVersionType { get; set; }
+    public int ComparedVersionId { get; set; }
+    public ResourceContentVersionTypes ComparedVersionType { get; set; }
+    public ResourceContentStatus Status { get; set; }
```

Comment:
Not 100% on using the `ResourceContentStatus` to identify the status for the comparison but using a custom status seemed inconsistent.

Reply:
I'm not really sure what goes here. If it's against a MachineTranslation type it probably doesn't have a meaning, and snapshot to snapshot can join on those to get their status at the time the snapshot was created. So unless there's something I'm not thinking of, I would drop this.

Reply:
Okay, I can see dropping it. Below is just a note on how this works in theory for posterity.

From what Frank had described, I got the impression of wanting to know the similarity score between a resource at two different stages (i.e. from machine to published, from editor review to company review, etc.). 
Here is a list to infer the status of a resource comparison:
 - machine to Rcv = published 'complete' content
 - machine to snapshot = whatever status the snapshot is
 - snapshot to snapshot = pick the status you are concerned with
 - Rcv to snapshot = snapshot status